lib/web_console/middleware.rb

```diff
@@ -16,6 +16,8 @@ class Middleware
       this request hit doesn't store %{id} in memory.
     END
 
+    Mime::Type.register 'application/vnd.web-console.v2', :web_console
```

Comment:
I think we should move this to the `Request` class file, so everything it depends on is defined in that file.